clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -71,13 +71,21 @@ export class CopybookDownloadService implements vscode.Disposable {
             }
             return false;
         }
-        if (!members.includes(copybookProfile.copybook)) {
+        const remoteCopybook = CopybookDownloadService.getRemoteCopybookName(members, copybookProfile.copybook);
```

Comment:
Looks like this piece of code is for setting "quiet" flag.
Let's move this logic to appropriate private method f.e. "makeCopybookQuite()"